CHANGELOG.md

```diff
@@ -1,5 +1,47 @@
 # Change Log
 
+## 2.2
+
+This release is mostly a maintenance release which fixes bugs and regressions.
+It also unblocks significant future feature development.
+
+### Workaround removals
+
+- Remove old `force_bindless_texel_buffer` workaround with
+  a more correct and performant workaround.
+  Death Stranding and Cyberpunk 2077 (and probably other games as well) do the right thing by default without the hack now.
+- Remove old workaround for occlusion queries which was enabled for AC: Valhalla.
+  Improves performance in AC: Valhalla.
```

Comment:
Does it? I don't think Valhalla uses queries enough for this to make a difference.
I'd just bunch this up with the texel buffer workaround (name was `disable_query_optimization`).

Reply:
Fixed.